Lib/sysconfig.py

```diff
@@ -112,6 +112,11 @@ def _safe_realpath(path):
 # PC/VS7.1
 if os.name == "nt" and "\\pc\\v" in _PROJECT_BASE[-10:].lower():
     _PROJECT_BASE = _safe_realpath(os.path.join(_PROJECT_BASE, pardir, pardir))
+# PC/VS9.0/amd64
+if (os.name == "nt" 
+   and os.path.basename(os.path.dirname(os.path.dirname(_PROJECT_BASE))).lower() == "pc" 
+   and os.path.basename(os.path.dirname(_PROJECT_BASE)).lower().startswith("vs")):
```

Comment:
I tried to do exactly that, but it's wrong :-) It seems like VS7.1 used python(d_).exe in PC/VS7.1/python(_d).exe, whereas VS 9.0 uses python(_d).exe in PC/VS9.0/{amd64,x86}/python(_d).exe: one more subdirectory!